{
    "type": "bundle",
    "id": "bundle--a4e2108f-0f11-44fa-8cd3-394fb0d2b544",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--b96bbff6-8a63-47b4-a971-04755765373f",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2024-07-02T01:00:54.596761Z",
            "modified": "2024-07-02T01:00:54.596761Z",
            "name": "CVE-2024-39428",
            "description": "In trusty service, there is a possible out of bounds write due to a missing bounds check. This could lead to local denial of service with System execution privileges needed",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2024-39428"
                }
            ]
        }
    ]
}